src/components/CourseItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+
+const courseItem = (props) => {
+
+  let { course,loadCourseData} = props;
```

Comment:
As above formatting is off, please use yarn format before pushing.

Reply:
As above